app/javascript/article-form/articleForm.jsx

```diff
@@ -73,6 +74,7 @@ export class ArticleForm extends Component {
     super(props);
     const { article, version, siteLogo } = this.props;
     let { organizations } = this.props;
+    const { schedulingEnabled } = this.props;
```

Comment:
nit: I believe this could be part of line 75. cc @aitchiss in case I made a mistake.

```suggestion
    const { article, version, siteLogo, schedulingEnabled } = this.props;
    let { organizations } = this.props;
```